app/views/responses/answers/_single_read_only.html.erb

```diff
@@ -32,7 +32,7 @@
 
 <% else %>
 
-  <%# show search excerpt (highlights search terms) if exists for this questioning, else just show plain answer value %>
-  <%= content_tag(:div, excerpt_if_exists(@response, answer)) %>
+  <!-- Rely on simple_format to sanitize by default. -->
+  <%= content_tag(:div, simple_format(answer.value)) %>
```

Comment:
The `simple_format` comment comes directly from its old location in `app/helpers/answers_helper.rb`

Reply:
Oops I'll fix the comment `!--` syntax

Reply:
Looks good.